### PR TITLE
Dockerization

### PR DESCRIPTION
**Pull request by gouthamcp**
Dockerization-Image-Name-Updation-Category-API-Working

### CODE REVIEW

File: UI/GroceyStore_UI/grocery_store/src/History.js [hunk 1]
```diff
@@ -1,17 +1,113 @@
...
+            </TableHead>
+            <TableBody>
+              {rows.slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage).map((row) => {
+                return (
+                  <TableRow hover role="checkbox" tabIndex={-1} key={row.code}>
+                    {columns.map((column) => {
+                      const value = row[column.id];
+                      return (
+                        <TableCell 
+                        numeric component="a" href="/Notification"
```
Comment (shariquemaaz) [inline UI/GroceyStore_UI/grocery_store/src/History.js:90]: Why are we navigating to 'Notification' directly from history? Am i missing something? Even if we need to navigate, use react router

File: Backend/GroceryStore_Backend/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+
+## Docker File Creation
+
+dotnet publish -c release -o app/ .
+docker build --pull -t grocerystore_backend .
```
Comment (shariquemaaz) [inline Backend/GroceryStore_Backend/README.md:5]: I think the build should come first and then publish

File: Backend/GroceryStore_Backend/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+
+## Docker File Creation
+
```
Comment (shariquemaaz) [inline Backend/GroceryStore_Backend/README.md:3]: Add a nuget restore first, then build and then publish

**Review by shariquemaaz — CHANGES_REQUESTED** [review]
Please remove all dll files. Do a nuget restore in the docker file which will download all these dlls.

File: Backend/GroceryStore_Backend/README.md (hunk 2 above)
Comment (gouthamcp) [reply Backend/GroceryStore_Backend/README.md:5]: No, this is correct step . Please do check out this link  [https://medium.com/faun/deploy-dotnet-core-api-docker-container-with-mysql-on-heroku-ed387eab4222](url)

File: Backend/GroceryStore_Backend/README.md (hunk 3 above)
Comment (gouthamcp) [reply Backend/GroceryStore_Backend/README.md:3]: okay i will

File: UI/GroceyStore_UI/grocery_store/src/History.js (hunk 1 above)
Comment (gouthamcp) [reply UI/GroceyStore_UI/grocery_store/src/History.js:90]: I used it for testing purpose.

**Review by shariquemaaz — APPROVED** [review]

**Pull request merged**